sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -133,12 +133,36 @@ class AstBuilder extends DataTypeAstBuilder with SQLConfHelper with Logging {
 
   private def visitCompoundBodyImpl(
       ctx: CompoundBodyContext,
-      label: Option[String]): CompoundBody = {
+      label: Option[String],
+      allowPrefixDeclare: Boolean): CompoundBody = {
```

Comment:
not sure if the `allowPrefixDeclare` is the proper name, it's unclear what "prefix" means... maybe just `allowVarDeclare`?